pyproject.toml

```diff
@@ -59,3 +59,97 @@ build-backend = "poetry_dynamic_versioning.backend"
 enable = true
 vcs = "git"
 style = "semver"
+
+[tool.ruff]
+line-length = 88
+exclude = [".git", "sktime/_contrib/*", "examples/blog_posts/*"]
```

Comment:
@fkiraly i think `sktime/*` is irrelevant to us here?

Reply:
yes, removed